integration/airflow/openlineage/lineage_backend/__init__.py

```diff
@@ -51,7 +51,7 @@ def send_lineage(
                 run_id=run_id,
                 job_name=job_name,
                 job_description=dag.description,
-                event_time=DagUtils.get_start_time(dagrun.execution_date),
+                event_time=DagUtils.get_start_time(task_instance.start_date),
```

Comment:
Is there a way we can test the fix?

Reply:
I think one reasonable approach would be to update our Airflow integration test, see [`test_integration.py`](https://github.com/OpenLineage/OpenLineage/blob/main/integration/airflow/tests/integration/test_integration.py). For each DAG run, we call [`wait_for_dag()`](https://github.com/OpenLineage/OpenLineage/blob/main/integration/airflow/tests/integration/test_integration.py#L39) with the DAG ID that queries the `dag_run` table. You can do something similar by checking the start time, nominal times, etc for each DAG run on completion:

```python
check_dag_start_time(DAG_ID)
check_dag_nominal_start_time(DAG_ID)
.
.
```

Reply:
Thank you for the feedback. I've added a check to the integration test case, simply checking the values and orders of `eventTime` that is actually generated. The issue was not on the DAG information but was the `eventTime` of the openlineage event, so checking the generated event time would be sufficient.